### PR TITLE
feat: aex9 balances for an account with token info

### LINKED ISSUE #416: Account list AEX9 Name obtained
Getting AEX9 account list does not know the token name directly

https://mainnet.aeternity.io/mdw/aex9/balances/account/ak_2kxt6D65giv4yNt4oa44SjW4jEXfoHMviPFvAreSEXvz25Q3QQ

### PR DESCRIPTION
**Pull request by jyeshe**
## What

Adds token info to `/aex9/balances/account/:account` response.

## Why

Resolves #416

## Validation steps

`elixir --sname aeternity@localhost -S mix test.integration test/integration/ae_mdw_web/controllers/aex9_controller_test.exs:203`
`elixir --sname aeternity@localhost -S mix test.integration test/integration/ae_mdw_web/controllers/aex9_controller_test.exs`

### CODE REVIEW

File: priv/migrations/20211224162001_reindex_aex9_account_presence.ex [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    {:atomic, pairs_to_update} = :mnesia.transaction(fn ->
+      Model.Aex9AccountPresence
+      |> :mnesia.all_keys()
+      |> Enum.into(MapSet.new(), fn {account_pk, _txi, contract_pk} ->
+        {account_pk, contract_pk}
+      end)
+    end)
+
+    indexed_count = :mnesia.sync_dirty(fn ->
```
Comment (jyeshe) [inline priv/migrations/20211224162001_reindex_aex9_account_presence.ex:27]: `:mnesia.transaction` takes more than 10 minutes
`:mnesia.sync_dirty` takes less than a minute

This difference might be helpful after having idempotent chain tx write.

File: priv/migrations/20211224162001_reindex_aex9_account_presence.ex [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      Enum.reduce(pairs_to_update, 0, fn {account_pk, contract_pk}, acc ->
+        contract_id = :aeser_api_encoder.encode(:contract_pubkey, contract_pk)
+        create_txi = Origin.tx_index({:contract, contract_id})
+
+        Contract.aex9_delete_presence(contract_pk, -1, account_pk)
+        Contract.aex9_write_presence(contract_pk, create_txi, account_pk)
+
+        acc = acc + 1
+        if rem(acc, 100) == 0, do: :mnesia.dump_log()
```
Comment (jyeshe) [inline priv/migrations/20211224162001_reindex_aex9_account_presence.ex:36]: respect default db log size (increasing the log write buffer beyond gives a warning).

File: lib/ae_mdw/sync/async_tasks/update_aex9_presence.ex [hunk 3]
```diff
@@ -24,10 +25,19 @@ defmodule AeMdw.Sync.AsyncTasks.UpdateAex9Presence do
...
+
+    if create_txi do
+      :mnesia.sync_dirty(fn ->
+        Enum.each(amounts, fn {{:address, account_pk}, _amount} ->
+          Contract.aex9_write_new_presence(contract_pk, create_txi, account_pk)
+        end)
       end)
-    end)
+    else
+      Log.error("Missing create_txi for contract #{contract_id}")
```
Comment (sborrazas) [inline lib/ae_mdw/sync/async_tasks/update_aex9_presence.ex:38]: This isn't really an error. There's many contracts that don't have a create txi

**Review by sborrazas — CHANGES_REQUESTED** [review]

File: lib/ae_mdw/sync/async_tasks/update_aex9_presence.ex (hunk 3 above)
Comment (jyeshe) [reply lib/ae_mdw/sync/async_tasks/update_aex9_presence.ex:38]: For now aex9 contracts is only indexed after a `:contract_create_tx` (with a `create_txi`).
Good observation though that Chain.create and Chain.clone should index aex9 contracts too. 
Issue opened #425

Comment (jyeshe) [reply lib/ae_mdw/sync/async_tasks/update_aex9_presence.ex:38]: For now aex9 contracts are only indexed after a `:contract_create_tx` (with a `create_txi`).
Good observation though that Chain.create and Chain.clone should index aex9 contracts too. 
Issue opened #425

**Review by thepiwo — APPROVED** [review]

**Review by sborrazas — APPROVED** [review]

**Pull request merged**